Jenkinsfile

```diff
@@ -23,8 +36,8 @@ pipeline {
                 always {
                     junit testResults: '**/build/test-results/test/*.xml'
                     recordIssues tools: [
-                      javaDoc(),
-                      taskScanner(includePattern: '**/*.java,**/*.groovy,**/*.gradle,**/*.kts', lowTags: 'WIBNIF', normalTags: 'TODO, FIXME', highTags: 'ASAP')
+                      javaDoc(id:"javaDoc-$JAVA_VERSION", name: "JavaDoc $JAVA_VERSION"),
+                      taskScanner(id:"tasks-$JAVA_VERSION", name: "Tasks $JAVA_VERSION", includePattern: '**/*.java,**/*.groovy,**/*.gradle,**/*.kts', lowTags: 'WIBNIF', normalTags: 'TODO, FIXME', highTags: 'ASAP')
```

Comment:
taskScanner works purely on the source text so it's not really useful to include in the matrix. Doing it once would be sufficient.